README.md

```diff
@@ -49,6 +49,12 @@ Benefit of this is you dont have to build container everytime you make a change
 ## Developer local Endpoints
 
 * Post an application 
+ ##### you can post with or without proceeding types, example
```

Comment:
**[not required for this PR]** As spoken offline, we should probably introduce https://swagger.io/docs/ to the app so that we don't have to manually update the README every single time a new endpoint is created/changed.

I'll create a github issue/story in JIRA to try and address that.